### PR TITLE
BaSP-MR-38: Test super admins put delete

### PR DESCRIPTION
**Pull request by CavalloFede**
Added seed for super admins
Added test for PUT endpoint
Added test for DEL endpoint

### CODE REVIEW

**Review by Eliezer-Alberto — APPROVED** [review]
I think it is correct!..

File: src/tests/super-admins.test.js [hunk 1]
```diff
@@ -1 +1,124 @@
...
+    expect(response.body.data).toBeUndefined();
+  });
+  test('No body and no Id is send on the request, should return a status 400 and a message', async () => {
+    const response = await request(app).put('/api/super-admins/').send();
+    expect(response.status).toBe(400);
+    expect(response.body.error).toBeTruthy();
+    expect(response.body.message).toBeDefined();
+    expect(response.body.data).toBeUndefined();
+  });
+  test('No Id is send on the request, should return a status 400 and a message', async () => {
```
Comment (nailaghigo) [inline src/tests/super-admins.test.js:50]: I think here the message status code number should be 404
```suggestion
  test('No Id is send on the request, should return a status 404 and a message', async () => {
```

**Review by nailaghigo — APPROVED** [review]
Just one little change and it'll be ready to merge 🚀

File: src/tests/super-admins.test.js (hunk 1 above)
Comment (CavalloFede) [reply src/tests/super-admins.test.js]: fixed

**Pull request merged**